src/core/js/router.js

```diff
@@ -322,6 +322,9 @@ define([
             }
 
             Adapt.location._currentLocation = currentLocation;
+            
+            var locationModel = Adapt.findById(id);
+            var startControllerClass = (locationModel && locationModel.get('isStartPage')) ? locationModel.get('_htmlClassName') : '';
```

Comment:
This switch is unnecessary, as I explained in [this comment](https://github.com/adaptlearning/adapt_framework/issues/2392#issuecomment-483215043) adding classes through the `startController` won't add the classes permanently to the page.

```js
var htmlClassName = locationModel && locationModel.get('_htmlClassName') || '';
```